### ISSUE TITLE
[modx3] Plugin events not shown

### ISSUE THREAD

**Issue by Mark-H** [modxcms/revolution#14813]
### Summary

When viewing a plugin in the manager, the System Events tab shows an empty grid.

### Step to reproduce

Install MODX 3.x and edit a plugin.

### Observed behavior

The system events grid is empty with the following information in the log:

```
[2019-11-01 14:00:52] (ERROR @ core/vendor/xpdo/xpdo/src/xPDO/xPDO.php : 772) MODX\Revolution\mysql\modEvent::getSelectColumns() is not a valid static method.
[2019-11-01 14:00:52] (ERROR @ core/vendor/xpdo/xpdo/src/xPDO/xPDO.php : 772) MODX\Revolution\mysql\modEvent::getSelectColumns() is not a valid static method.
[2019-11-01 14:00:52] (ERROR @ core/vendor/xpdo/xpdo/src/xPDO/xPDO.php : 772) MODX\Revolution\mysql\modEvent::loadCollection() is not a valid static method.
[2019-11-01 14:00:52] (ERROR @ core/src/Revolution/Processors/Element/Plugin/Event/GetList.php : 49) PHP warning: Invalid argument supplied for foreach()
```

I've traced the code back to core/src/Revolution/mysql/modEvent.php `listEvents` method but I'm not quite sure why that would not be working. Seems pretty standard code.

### Expected behavior

Would be nice of events were listed. ;) The regular system event listing (in System > System Settings > System Events tab) does work as expected. 

### Environment

3.x @ d26ff17481441ddb3d1c596c52c920f3e3bb2984. Seems unrelated to specific environment.

**Comment by Ruslan-Aleev** [comment]
Yes, I confirm the bug, although several PRs ago it was not observed.

**Comment by theboxer** [comment]
hh, it's caused by using wrong class `MODX\Revolution\mysql\modEvent` instead of `\MODX\Revolution\modEvent`

**Comment by Ibochkarev** [comment]
Fixed in #14833 
@modxbot close
